--- origsrc/grcompiler-4.2/compiler/Makefile.am	2010-09-13 21:49:05.000000000 -0500
+++ src/grcompiler-4.2/compiler/Makefile.am	2012-09-10 23:28:10.816664700 -0500
@@ -1,7 +1,7 @@
 AM_CPPFLAGS = -I@srcdir@/Generic -I@srcdir@/Grammar
 
 SUBDIRS = Generic Grammar
-LIBS = @LIBS@ @LIBICONV@ -LGeneric -LGrammar -lgeneric -lparser
+LIBS = -LGeneric -LGrammar -lgeneric -lparser @LIBS@ @LIBICONV@
 EXTRA_DIST = resource.h GrCompiler.rc GrpParser.g GrpParser_readme.txt GrpParserTokenTypes.txt stddef.gdh
 
 bin_PROGRAMS = grcompiler
